pyproject.toml

```diff
@@ -33,7 +33,7 @@ python = "^3.8"
 
 poetry-core = "1.9.0"
 poetry-plugin-export = "^1.7.0"
-build = "^1.0.3"
+build = "^1.1"
```

Comment:
Shall we require `"^1.1.1"` because `1.1.0` is yanked?

Reply:
Sounds reasonable